core/ecschema-editing/src/Editing/Entities.ts

```diff
@@ -70,9 +70,6 @@ export class Entities extends ECClasses {
 
   public async create(schemaKey: SchemaKey, name: string, modifier: ECClassModifier, displayLabel?: string, baseClassKey?: SchemaItemKey, mixins?: Mixin[]): Promise<SchemaItemEditResults> {
     const schema = await this._schemaEditor.getSchema(schemaKey);
-    if (schema === undefined)
```

Comment:
this seems like a good check

Reply:
The getSchema method throws an exception. I suppose it doesn't hurt to have it, but the condition would never be true in practice.